pt-BR/mail/chrome/messenger/messengercompose/messengercompose.dtd

```diff
@@ -206,40 +206,40 @@
 <!ENTITY attachButton.label "Anexar">
 <!ENTITY spellingButton.label "Ortografia">
 <!ENTITY saveButton.label "Salvar">
-<!ENTITY tagButton.label "Assign Topic">
-<!ENTITY throbberItem.title "Activity Indicator">
-<!ENTITY signatureButton.label "Signatures">
-<!ENTITY compose.sidebar.label "Sidebar">
+<!ENTITY tagButton.label "Atribuir tópico">
+<!ENTITY throbberItem.title "Indicador de atividade">
+<!ENTITY signatureButton.label "Assinaturas">
+<!ENTITY compose.sidebar.label "Barra lateral">
 
 <!-- Mail Toolbar Tooltips -->
-<!ENTITY sendButton.tooltip "Enviar esta mensagem agora"> 
-<!ENTITY sendlaterButton.tooltip "Armazenar a mensagem na pasta “Saída”"> 
-<!ENTITY quoteButton.tooltip "Citar a mensagem anterior"> 
-<!ENTITY addressButton.tooltip "Selecionar destinatários no catálogo de endereços"> 
-<!ENTITY attachButton.tooltip "Anexar arquivos"> 
-<!ENTITY saveButton.tooltip "Salvar a mensagem na pasta “Rascunhos”"> 
+<!ENTITY sendButton.tooltip "Enviar esta mensagem agora">
+<!ENTITY sendlaterButton.tooltip "Armazenar a mensagem na pasta “Saída”">
+<!ENTITY quoteButton.tooltip "Citar a mensagem anterior">
+<!ENTITY addressButton.tooltip "Selecionar destinatários no catálogo de endereços">
```

Comment:
I disagree, and note that this is not my translation (only trailing spaces were removed).